### PR TITLE
chore(store): translate CJK comments to English for store-dist, store-grpc, store-node, store-rocksdb, store-test

### PR DESCRIPTION
**Pull request by VGalaxies**
subtask of https://github.com/apache/incubator-hugegraph/issues/2575

Except for some Chinese character tests in https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/util/UnsafeUtilTest.java.

### CODE REVIEW

File: hugegraph-store/hg-store-grpc/pom.xml
Comment (VGalaxies) [inline hugegraph-store/hg-store-grpc/pom.xml:94]:
```suggestion
                    <!-- Default value -->
```

File: hugegraph-store/hg-store-grpc/pom.xml
Comment (VGalaxies) [inline hugegraph-store/hg-store-grpc/pom.xml:96]:
```suggestion
                    <!-- Default value -->
```

File: hugegraph-store/hg-store-grpc/pom.xml
Comment (VGalaxies) [inline hugegraph-store/hg-store-grpc/pom.xml:99]:
```suggestion
                    <!-- Set whether to clear the files in outputDirectory before generating java files, the default value is true, and it will also override files with the same name when set to false -->
```

File: hugegraph-store/hg-store-grpc/pom.xml
Comment (VGalaxies) [inline hugegraph-store/hg-store-grpc/pom.xml:101]:
```suggestion
                    <!-- More configuration information can be found at https://www.xolstice.org/protobuf-maven-plugin/compile-mojo.html -->
```

File: hugegraph-store/hg-store-grpc/pom.xml
Comment (VGalaxies) [inline hugegraph-store/hg-store-grpc/pom.xml:105]:
```suggestion
                        <!-- When executing mvn compile, the following operations will be performed -->
```

File: hugegraph-store/hg-store-grpc/pom.xml
Comment (VGalaxies) [inline hugegraph-store/hg-store-grpc/pom.xml:108]:
```suggestion
                            <!-- Generate OuterClass class -->
```

File: hugegraph-store/hg-store-grpc/pom.xml
Comment (VGalaxies) [inline hugegraph-store/hg-store-grpc/pom.xml:110]:
```suggestion
                            <!-- Generate Grpc classes -->
```

File: hugegraph-store/hg-store-grpc/src/main/proto/graphpb.proto
Comment (VGalaxies) [inline hugegraph-store/hg-store-grpc/src/main/proto/graphpb.proto:79]:
```suggestion
  repeated Property properties = 3; // Point properties
```

File: hugegraph-store/hg-store-grpc/src/main/proto/graphpb.proto
Comment (VGalaxies) [inline hugegraph-store/hg-store-grpc/src/main/proto/graphpb.proto:89]:
```suggestion
  repeated Property properties = 6; // Edge properties
```

File: hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/grpc/HgStoreNodeService.java
Comment (VGalaxies) [inline hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/grpc/HgStoreNodeService.java:143]:
```suggestion
            // Serialization
```

File: hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/grpc/HgStoreNodeService.java
Comment (VGalaxies) [inline hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/grpc/HgStoreNodeService.java:150]:
```suggestion
            // Send to raft
```

File: hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/grpc/ParallelScanIterator.java
Comment (VGalaxies) [inline hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/grpc/ParallelScanIterator.java:110]:
```suggestion
                    current = queue.poll(100, TimeUnit.MILLISECONDS);  // Regularly check if the client has been closed.
```

File: hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/HgSessionManagerOneRaftFakePDTest.java
Comment (VGalaxies) [inline hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/HgSessionManagerOneRaftFakePDTest.java:177]:
```suggestion
    // CAUTION: ONLY FOR LONG!
```

File: hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/HgSessionManagerRaftFakePDTest.java
Comment (VGalaxies) [inline hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/HgSessionManagerRaftFakePDTest.java:219]:
```suggestion
    // CAUTION: ONLY FOR LONG!
```

File: hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/HgSessionManagerRaftPDTest.java
Comment (VGalaxies) [inline hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/HgSessionManagerRaftPDTest.java:265]:
```suggestion
    // CAUTION: ONLY FOR LONG!
```

File: hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/HgSessionManagerTest.java
Comment (VGalaxies) [inline hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/HgSessionManagerTest.java:175]:
```suggestion
    // CAUTION: ONLY FOR LONG!
```

File: hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/PartitionEngineTest.java
Comment (VGalaxies) [inline hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/PartitionEngineTest.java:91]:
```suggestion
        // Delete 4, 5
```

File: hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/client/HgSessionManagerRaftPDTest.java
Comment (VGalaxies) [inline hugegraph-store/hg-store-test/src/main/java/org/apache/hugegraph/store/client/HgSessionManagerRaftPDTest.java:243]:
```suggestion
    // CAUTION: ONLY FOR LONG!
```

File: hugegraph-store/hg-store-test/src/main/resources/pd-server.yml
Comment (VGalaxies) [inline hugegraph-store/hg-store-test/src/main/resources/pd-server.yml:70]:
```suggestion
  # After client registration, no heartbeat longest number, after exceeding, previous registration information will be deleted.
```

**Review by Pengzna — APPROVED** [review]
LGTM!!

image

**Review by imbajin — APPROVED** [review]

File: hugegraph-store/hg-store-rocksdb/src/main/java/org/apache/hugegraph/rocksdb/access/RocksDBFactory.java
Comment (JackyYangPassion) [inline hugegraph-store/hg-store-rocksdb/src/main/java/org/apache/hugegraph/rocksdb/access/RocksDBFactory.java:216]: map ---> Graph

**Review by JackyYangPassion — CHANGES_REQUESTED** [review]

File: .asf.yaml
Comment (javeme) [inline .asf.yaml:31]: Prefer to disable merge button.
In order to keep a clear history, we do not expect to introduce merge commits.

**Review by javeme — CHANGES_REQUESTED** [review]

File: .asf.yaml
Comment (VGalaxies) [reply .asf.yaml:31]: Temporarily enable for migrating commons module, disable it after merge #2628... cc @imbajin

Comment (imbajin) [reply .asf.yaml:31]: > Temporarily enable for migrating commons module, disable it after merge #2628... cc @imbajin

Yes, just temporarily open, then close again

**Review by imbajin — DISMISSED** [review]

**Review by JackyYangPassion — APPROVED** [review]

File: .asf.yaml
Comment (javeme) [reply .asf.yaml:31]: get it

Comment (imbajin) [reply .asf.yaml:31]: > get it

@javeme need your approve to merge this PR 🔢

**Review by imbajin — DISMISSED** [review]

**Review by imbajin — APPROVED** [review]

**Comment by VGalaxies** [comment]
@javeme need your approval to merge this PR 😶‍🌫️

**Pull request closed without merging**
